portfolio/src/main/java/com/google/sps/data/Comment.java

```diff
@@ -18,17 +18,23 @@
 
 public final class Comment {
 
-  private final Date timestamp;
+  private final long id;
```

Comment:
Please add a docstring above describing this class.

Reply:
Please reformat to match the standard for multiline javadocs: https://google.github.io/styleguide/javaguide.html#s7.1.1-javadoc-multi-line

Reply:
Got it